storage/bootstrap/types.go

```diff
@@ -50,34 +72,169 @@ type ShardResult interface {
 	// RemoveSeries removes a single series of blocks.
 	RemoveSeries(id string)
 
-	// GetAllSeries returns all series of blocks.
-	GetAllSeries() map[string]block.DatabaseSeriesBlocks
-
 	// Close closes a shard result.
 	Close()
 }
 
+// ShardResults is a map of shards to shard results.
+type ShardResults map[uint32]ShardResult
+
+// AddResults adds other shard results to the current shard results.
+func (r ShardResults) AddResults(other ShardResults) {
```

Comment:
Move the implementation to a separate file (e.g., `result.go`)? `types.go` should only contain types